packages/cli/src/lib/bundler/bundle.ts

```diff
@@ -47,18 +47,44 @@ export async function buildBundle(options: BuildOptions) {
     targetPath: paths.targetPath,
   });
 
-  const config = await createConfig(paths, {
+  const commonConfigOptions = {
     ...options,
     checksEnabled: false,
     isDev: false,
     baseUrl: resolveBaseUrl(options.frontendConfig),
     getFrontendAppConfigs: () => options.frontendAppConfigs,
-    additionalEntryPoints: detectedModulesEntryPoint,
+  };
+  const configs = [
+    await createConfig(paths, {
+      ...commonConfigOptions,
+      additionalEntryPoints: detectedModulesEntryPoint,
+    }),
+  ];
+
+  const publicPaths = await resolveOptionalBundlingPaths({
+    entry: 'src/index-public-experimental',
+    dist: 'dist/public',
   });
+  if (publicPaths) {
+    console.log(
+      chalk.yellow(
+        `⚠️  WARNING: The app /auth entry point is an experimental feature that may receive immediate breaking changes.`,
```

Comment:
@Rugvip it looks like in f518683 you changed the path to `/public` but this warning still mentions `/auth` -- maybe just missed updating the log message? Just want to make sure I'm correctly understanding what's going on here.

Reply:
Yep, thank you! 👍